src/views/queries/queriesList.tsx

```diff
@@ -63,6 +65,28 @@ export default class QueryListTemplates {
         return <p>not sure</p>
     }
   }
+  private buttonText = (status: string | QueryStatus, role: QueryRole) => {
+    if (status === 'resolved' && role === 'requester') {
+      return 'View Response'
+    } else if (status === 'pending_their_input' && role === 'requester') {
+      return 'Awaiting Response'
+    } else if (status === 'pending_your_input' && role === 'responder') {
+      return 'Respond to query'
+    } else if (status === 'resolved' && role === 'responder') {
+      return 'no action required'
+    }
+    return 'not sure'
```

Comment:
I think this should remain/be renamed because there is other cases like status 'errored' which is valid we would not want to just blatantly throw I think

Reply:
yea, it makes more sense `no action available`